packages/sampler/stories/qa-stories/components/ScrollerResizableItem/ScrollerResizableItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@enact/moonstone/Button';
+
+
+class ScrollerResizableItem extends React.Component {
+
+	render () {
+		const height = this.props.more ? 1500 : 400;
+		const text = this.props.more ? 'less' : 'more';
+		return (
+			<div style={{position: 'relative', width: '90%', height: height, border: 'solid 3px yellow'}}>
```

Comment:
3px -> ri.unit(3, 'rem')

Reply:
How about making css object except for height?
const css = {
    position: 'relative',
    ...
};

Reply:
How about...

```
	render () {
		const height = ri.unit(this.props.more ? 1500 : 400, 'rem');
		const text = this.props.more ? 'less' : 'more';
		const style = {
			position: 'relative',
			width: '90%',
			border: 'solid ' + ri.unit(3, 'rem') + ' yellow'
		};

		return (
			<div style={{...style, height}}>
				<Button onClick={this.props.toggleMore} small style={{position: 'absolute', bottom: 0}}>{text}</Button>
			</div>
		);
	}
```

In addition, I fixed `const height = ri.unit(this.props.more ? 1500 : 400, 'rem');` code with `rem`.

Reply:
That's good idea. I've removed `3px` because I checked default value of `border-width` is `3px`.